src/Core/Components/List/FluentPersona.razor

```diff
@@ -1,23 +1,23 @@
-@namespace Microsoft.FluentUI.AspNetCore.Components
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
 @inherits FluentComponentBase
 
 <div id="@Id" class="@ClassValue" style="@StyleValue">
     <div class="initials">
         <FluentPresenceBadge Status="@Status" Size="@StatusSize" StatusTitle="@StatusTitle">
-            @if (string.IsNullOrEmpty(Image))
-            {
-                <div style="display: table-cell; vertical-align: middle; @GetImageSizeStyle()">
-                        @(Initials ?? GetDefaultInitials())
-                </div>
-            }
-            else
-            {
-                <img src="@Image" alt="@Name" title="@Name" style="@GetImageSizeStyle()" />
-            }
+            <div style="display: table-cell; vertical-align: middle; @GetImageSizeStyle()">
+                @if (string.IsNullOrEmpty(Image))
+                {
+                    @(Initials ?? GetDefaultInitials())
+                }
+                else
+                {
+                    <img src="@Image" alt="@Name" title="@Name" style="@($"max-width: {ImageSize}; max-height: {ImageSize}")" />
```

Comment:
`ImageSize` could be null. So, you need to use `GetImageSizeStyle`.

Reply:
Well, I'm on vacation the next two weeks 😅 I can change it afterwards, but otherwise, feel free to change anything to your liking